WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -214,14 +214,17 @@ public static async Task<RpcStatus> GetRpcStatusAsync(this IRPCClient rpc, Cance
 			return newEstimations;
 		}
 
-		public static async Task<(bool accept, string rejectReason)> TestMempoolAcceptAsync(this IRPCClient rpc, IEnumerable<Coin> coins)
+		public static async Task<(bool accept, string rejectReason)> TestMempoolAcceptAsync(this IRPCClient rpc, IEnumerable<Coin> coins, int fakeOutputsCount = 1)
```

Comment:
`= 1` default value makes me suspicious that you didn't want to implement it to all relevant places where `TestMempoolAcceptAsync` is used. Can you confirm that you use it in every relevant places?

Reply:
There is only one other place:

https://github.com/molnard/WalletWasabi/blob/067f038d5a1c005e258ae051f872c2fdc884b59b/WalletWasabi/CoinJoin/Coordinator/Rounds/CoordinatorRound.cs#L1186

I did not touch it yet. Now we are using Alices one by one - so in the end, Wasabi will remove the rejected ones. Do you have a better idea of how to determine which Alice to remove in a more reliable way?

Reply:
For example, I can generate CoinJoins with every input except one the current Alice and try to pick a successful combination. If there is none then remove one Alice and start to create the combinations again. 
I do not want to do this as it increases the complexity and I am not sure how long does it take for the node to calculate this.